### ISSUE TITLE
Cookies with 'expires' values in the past aren't deleted properly and are still sent with requests

### ISSUE THREAD

**Issue by andrewh-agl** [DevExpress/testcafe#7432]
### What is your Scenario?

Cookies deleted using an 'expires' date in the past, without setting any 'max-age' property, aren't deleted properly and are still sent with requests.

This is almost exactly the same issue to the one recently fixed in https://github.com/DevExpress/testcafe-hammerhead/pull/2818, but that PR focused on cookies that used only 'max-age', rather than only 'expires'. 

### What is the Current behavior?

We use a 3rd party Javascript library on our website that deletes its cookies using past dated expiry values like this:

```JavaScript
document.cookie = "MY_COOKIE=; path=/; expires=Sun, 18 Dec 2022 22:53:55 GMT;" // date is in the past
```
They do not set any 'max-age' values.

Testcafe is not deleting these past dated cookies and is still sending them with http requests

### What is the Expected behavior?

Cookies with past dated expiry values should be deleted and not sent with http requests even if the cookie is not set with a max-age property

### What is your public website URL? (or attach your complete example)

I used localhost - see 'Steps to Reproduce'

### What is your TestCafe test code?

see: [cookie.test.js](https://github.com/andrewh-agl/testcafe-cookie-test/blob/main/test/cookie.test.js)

### Your complete test report

✖ expect a deleted cookie to be gone

   1) AssertionError: expected 'MY_COOKIE=' to deeply equal ''
      
      + expected - actual
      
      -MY_COOKIE=
      +''
      

      Browser: Chrome 108.0.0.0 / macOS 10.15.7

         13 |  await t.navigateTo("/");
         14 |  await t.expect(await getCookie()).eql("MY_COOKIE=Hello");
         15 |
         16 |  await t.click(clearCookieButton);
         17 |  await t.navigateTo("/");
       > 18 |  await t.expect(await getCookie()).eql("");
         19 |}).disablePageCaching;

### Steps to Reproduce

1. git clone https://github.com/andrewh-agl/testcafe-cookie-test
2. run `npm install` and `npm run serve` in one terminal
3. run `npm run test` in a second terminal

The crux of the issue is the deletion of the cookie using a past dated expiry in the [index.html](https://github.com/andrewh-agl/testcafe-cookie-test/blob/main/website/index.html#L17)

### TestCafe version

2.2.0-rc.1

### Node.js version

v14.18.1

### Command-line arguments

chrome test/cookie.test.js

### Browser name(s) and version(s)

Chrome 108

### Platform(s) and version(s)

macOS 12.6

**Comment by Aleksey28** [comment]
Hi @andrewh-agl,

There is a [PR with a fix](https://github.com/DevExpress/testcafe-hammerhead/pull/2830). It will be merged and published in the nearest release after 2.2.0.

**Comment by andrewh-agl** [comment]
Thanks @Aleksey28 for addressing this issue, I'll try it out when the new release is published
